src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js

```diff
@@ -1393,53 +1438,85 @@ export default compose(
                 {
                   content: {
                     field: fieldName,
-                    value: [`${parseBucketValue(key)}`],
+                    value: [keyMin],
                   },
                   op: '>=',
                 },
-                ...(acc.nextInterval !== 0 ? [
+                [
                   {
                     content: {
                       field: fieldName,
-                      value: [`${parseBucketValue(acc.nextInterval - 1)}`],
+                      value: [keyMax],
                     },
                     op: '<=',
                   },
-                ] : []),
+                ],
               ],
               op: 'and',
-            };
+            } : {};
 
-        return {
-          data: [
-            ...acc.data,
-            {
-              chart_doc_count: doc_count,
-              doc_count: getCountLink({
-                doc_count,
-                filters,
-                totalDocs,
-              }),
+        return [
+          ...acc,
+          {
+            chart_doc_count: doc_count,
+            doc_count: getCountLink({
+              doc_count,
               filters,
-              key: getRangeValue(key, acc.nextInterval),
-              rangeValues: {
-                max: Math.floor(acc.nextInterval - 1),
-                min: key,
-              },
+              totalDocs,
+            }),
+            filters,
+            key: getContinuousRangeValue(keyValues),
+            rangeValues: {
+              max: keyMax,
+              min: keyMin,
             },
-          ],
-          nextInterval: key,
-        };
+          },
+        ];
       },
     })
   ),
+  withProps(({ data, fieldName, variable }) => {
```

Comment:
create default continuous data for

- resetting buckets
- showing default information in the continuous binning modal